src/site/content/en/fast/signed-exchanges-webpackager/index.md

```diff
@@ -441,9 +441,9 @@ used with signed exchanges.
    following changes:
     *   Update the line `PEMFile = cert.pem` to reflect the location of your PEM file.
     *   (Optional) To have `webpkgserver` auto-renew the SXG certificate every
-        90 days, in the `[SXG.ACME]` section of `webpkgserver.toml`, change
-        `#Enable = false` to `Enable = true`. This option only applies to sites
-        with a DigiCert ACME account setup.
+        90 days, configure the options in the `[SXG.ACME]` section of
```

Comment:
@twifkak FYI I changed this - I think this is more accurate?

Reply:
Yeah, good call.